x-pack/plugins/reporting/public/lib/reporting_api_client/reporting_api_client.test.ts

```diff
@@ -293,21 +287,15 @@ describe('ReportingAPIClient', () => {
     it('should send a post request', async () => {
       await apiClient.verifyBrowser();
 
-      expect(httpClient.post).toHaveBeenCalledWith(
-        expect.stringContaining('/diagnose/browser'),
-        expect.any(Object)
-      );
+      expect(httpClient.post).toHaveBeenCalledWith(expect.stringContaining('/diagnose/browser'));
     });
   });
 
   describe('verifyScreenCapture', () => {
     it('should send a post request', async () => {
       await apiClient.verifyScreenCapture();
 
-      expect(httpClient.post).toHaveBeenCalledWith(
-        expect.stringContaining('/diagnose/screenshot'),
-        expect.any(Object)
```

Comment:
the options object was removed from the API calls for many of these, since they mistakenly were used to send the `asSystemRequest` field